commitlintplugins.test.ts

```diff
@@ -304,3 +304,17 @@ test('type-space-before-paren3', () => {
     let typeSpaceBeforeParen3 = runCommitLintOnMsg(commitMsgWithNoSpaceBeforeParen);
     expect(typeSpaceBeforeParen3.status).toBe(0);
 });
+
+
+test('type-with-square-brackets1', () => {
+    let commitMsgWithSquareBracketStyle = "[foo] this is a title";
+    let rejectSquareBracketStyle1 = runCommitLintOnMsg(commitMsgWithSquareBracketStyle);
+    expect(rejectSquareBracketStyle1.status).not.toBe(0);
+});
+
+
+test('type-with-square-brackets2', () => {
+    let commitMsgWithoutSquareBracketStyle = "foo: this is a title";
+    let rejectSquareBracketStyle2 = runCommitLintOnMsg(commitMsgWithoutSquareBracketStyle);
```

Comment:
@realmarv renaming a rule has many side-effects sorry, these vars need to be renamed too

Reply:
Done